src/main/java/com/odysseusinc/arachne/portal/service/impl/BasePaperServiceImpl.java

```diff
@@ -465,6 +465,7 @@ public void fullDelete(List<P> papers) {
     }
 
     @Override
+    @Transactional
```

Comment:
Fixing anything else?

Reply:
Yes, this fixes an error during paper indexing.